test/io/jesi/backpack/macros_test.cljc

```diff
@@ -69,23 +70,50 @@
 (deftest shorthand-test
 
   #?(:clj (testing "is a macro"
-            (is (:macro (meta #'shorthand)))))
+            (is (macro? `shorthand))))
 
   (testing "creates a map with the keywords from the symbol names"
     (let [a 1
           b 2
           c {:cheese false}]
       (is= {:a 1} (shorthand a))
       (is= {:b 2} (shorthand b))
-      (is= {:a 1 :b 2} (shorthand a b))
+      (is= {:a 1, :b 2} (shorthand a b))
```

Comment:
```suggestion
      (is= {:a 1 :b 2} (shorthand a b))
```
What's this comma business...

Reply:
I find the comma more readable when the map is on a single line

Reply:
We don't use it anywhere else. If it's more readable, then it needs to become a standard and used across projects. i.e. A coding convention change.

You're call:
* Remove Commas
* Update Clojure Convention

Reply:
I'll update our coding convention

Reply:
https://github.com/jesims/docs/pull/108